channels/chan_usbradio.c

```diff
@@ -2031,6 +2045,14 @@ static struct ast_frame *usbradio_read(struct ast_channel *c)
 	if (o->readerrs) {
 		ast_log(LOG_WARNING, "USB read channel [%s] was not stuck.\n", o->name);
 	}
+
+	/* TBR - below appears to be an attempt to match levels to the original CM108
+	 * IC which has been out of production for over 10 years. Scaling all rx audio to 
+	 * 80% would result in a 20% loss in dynamic range, added quantization noise, and 
+	 * outgoing IAX audio reduced by 2dB. Any adjustments for CM1xxx IC gain differences
+	 * should be made in the mixer settings, not in the audio stream itself.
+	 */
+#if 0
```

Comment:
Please see my comments in chan_simpleusb.c re: concerns about this #if ... #endif block.

Reply:
Bug report opened: https://github.com/AllStarLink/app_rpt/issues/399